tdameritrade/auth/__init__.py

```diff
@@ -1,40 +1,17 @@
 import os
 import os.path
-import sys
 import time
 import urllib.parse as up
-from shutil import which
-
 import requests
 
 
 def authentication(client_id, redirect_uri, tdauser=None, tdapass=None):
     from selenium import webdriver
+    from webdriver_manager.chrome import ChromeDriverManager
```

Comment:
should add to setup.py requirements

Reply:
On the second thought, it would be nice to at least support both Chrome and Firefox. Thanks.

Reply:
open to making it an arg, something like `browser='chrome'`